Rules/CTF/Scripts/CTF_GiveSpawnItems.as

```diff
@@ -9,7 +9,7 @@ const u32 materials_wait_warmup = 40; //seconds between free mats
 //property
 const string SPAWN_ITEMS_TIMER = "CTF SpawnItems:";
 
-string base_name() { return "tent"; }
+const string[] BASE_NAME = {"tent", "warboat", "ballista"};
```

Comment:
All blobs that the player can respawn from are tagged with `"respawn"`. `BASE_NAME` can be replaced with a check to see if overlapping blobs have the tag.